qiskit/circuit/quantumcircuit.py

```diff
@@ -2229,7 +2437,43 @@ def qubits(self) -> list[Qubit]:
     @property
     def clbits(self) -> list[Clbit]:
         """A list of :class:`Clbit`\\ s in the order that they were added.  You should not mutate
-        this."""
+        this.
+
+        Example:
+
+            .. plot::
+                :include-source:
+                :nofigs:
+                :context: reset
+
+                from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+
+                qr1 = QuantumRegister(2)
+                qr2 = QuantumRegister(1)
+                cr1 = ClassicalRegister(2)
+                cr2 = ClassicalRegister(1)
+                qc = QuantumCircuit(qr1, qr2, cr1, cr2)
+
+                print("List the quantum registers:", qc.qregs)
+                print("List the classical registers:", qc.cregs)
+                print("List the qubits in this circuit:", qc.qubits)
+                print("List the ancilla qubits:", qc.ancillas)
+                print("List the classical bits in this circuit:", qc.clbits)
+
+            .. code-block:: text
+
+                List the quantum registers: [QuantumRegister(2, 'q0'),
+                QuantumRegister(1, 'q1')]
+                List the classical registers: [ClassicalRegister(2, 'c0'),
+                ClassicalRegister(1, 'c1')]
+                List the qubits in this circuit: [Qubit(QuantumRegister(2,
+                'q0'), 0), Qubit(QuantumRegister(2, 'q0'), 1), Qubit
+                (QuantumRegister(1, 'q1'), 0)]
+                List the ancilla qubits: []
+                List the classical bits in this circuit: [Clbit
+                (ClassicalRegister(2, 'c0'), 0), Clbit(ClassicalRegister(2,
+                'c0'), 1), Clbit(ClassicalRegister(1, 'c1'), 0)]
+        """
```

Comment:
Two things:

- I'm welcome to whatever the docs team decide, but personally I'd think that providing examples for even the most basic "property" getter methods like this might not actually be helping the reader - it's quite a lot of text to parse and so could make something that's really the most basic thing of Python programming (get a data attribute) look more complicated than it is.
- If we do keep it, the "text output" probably got its line wrappings in the place you copied it out from - we might want to both simplify the output (not print qregs, etc) and put line breaks in more logical places.

Reply:
shortened and tightened up!